PowerEditor/src/ScintillaComponent/FindReplaceDlg.h

```diff
@@ -232,6 +232,7 @@ class FindInFinderDlg : public StaticDialog
 private:
 	Finder  *_pFinder2Search = nullptr;
 	FindOption _options;
+	bool firstTimeInit = true;

```

Comment:
Instead of adding new member variable, I would 	change the value of `_options._isWholeWordand` & `_options._isMatchCase` to `false` in the constructor of FindInFinderDlg.

Reply:
You're right; it is simpler and thus better.